ddl/functions/get_user_score.sql

```diff
@@ -62,6 +63,18 @@ create or replace function get_user_score(target_user_id integer) returns table(
                 and u.is_verified = false then true
                 else false
             end as is_audius_impersonator,
+            case
+                when (
+                    exists (
+                        select 1
+                        from unnest(array['airdrop']) as badword
```

Comment:
so badly want a `badwords` table with naughtniess scores. But I _suppooooseeee_ this will do.